makefile

```diff
@@ -3,6 +3,8 @@ SHELL := /bin/bash
 .DEFAULT_GOAL := build
 
 TAG?=latest
+OS_IMAGE?=ArchLinuxARM-rpi-latest.tar.gz
+OS_URI?=http://archlinuxarm.org/os/$(OS_IMAGE)
```

Comment:
These are duplicated with `bin/load-os` not sure the best way to remove duplication.

Reply:
It's smart and helpful to have OS_IMAGE and OS_URI specified in the makefile, and in this case I feel we can keep the duplication as such. My thinking is that some users may wish to run the docker container without this specific makefile, for example from their own build script or as part of a larger toolchain.